### DISCUSSION TITLE
Default Cover Art Request

### DISCUSSION THREAD

**Discussion by R101G** [rern/rAudio#307] [Fixed Issues]
@rern,

Is there a way of changing the default cover art to the rAudio logo rather than the meter?

Also, having the album name displayed for the Paradise Radio feeds is great, but I am not always sure which feed I am listening to. Would it be possible to have the station name next to the webradio symbol?

Thanks.

**Comment by rern** [comment]
Try `UPDATE` branch.

**Reply by R101G** [reply]
Thanks rern, I now have the RP logo instead of the Webradio one, but nothing to say if it's Main or Mellow in landscape mode.
Should there be a way to change the default cover art now?

**Reply by rern** [reply]
WebRadio displays VU meter only when no station coverart available.
- Playback page - Long-press while stop > Coverart icon
- Library page list - Context menu > Change coverart

**Reply by R101G** [reply]
The problem is that I have a lot of miscellaneous files on a NAS, which have no cover art. The meter is very impressive work, but I would prefer to have a +R logo instead.

**Reply by EarsofTin** [reply]
I love the VU meter but maybe it's an age thing since I grew up with tape decks and meters.

**Reply by R101G** [reply]
@EarsofTin 

It's a great piece of artwork, but for me it's like having an interesting ornament which does not fit with the rest of the room. I am not saying it should go, I'd just like to have an alternative, and the project logo would seem to be a good choice.

**Comment by R101G** [comment]
@rern,

Thanks for the 20210428 update. It's just the ticket.
